packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/graphql/subscription/labelFragment.types.ts

```diff
@@ -18,6 +18,13 @@ export interface GQLLabel {
   style: GQLLabelStyle;
 }
 
+export interface GQLInsideLabel {
```

Comment:
There is no need for an alias Sprotty.
I removed the type `GQLInsideLabel`.

The alias does not fit in React flow since the inside label location will be introduced very soon.